docs/source/glossary.rst

```diff
@@ -71,11 +71,14 @@ Glossary
     Stamp
       The public key for a :term:`Character`'s signing key pair.
 
+    Treasure Map
+      The locations of :term:`Ursulas<Ursula>` that have the relevant :term:`kFrags<kFrag>` for a policy. :term:`Bob` will use the treasure map to determine which :term:`Ursulas<Ursula>` to contact when he wants to retrieve the data :term:`Alice` has shared with him.
```

Comment:
From this wording, it seems that Ursulas have the data that Bob needs. Perhaps just change "retrieve" by "decrypt"?

Reply:
✔️ 

Used `re-encrypt`